api.php

```diff
@@ -1381,6 +1322,10 @@ protected function load(){
 	public function __construct($route){
 		foreach (self::$permissionFields as $permissionField)
 			$this->permissions[$permissionField] = false;
+		
+		// every session user can get state of all games (i.e. spectate)
+		$this->permissions["getStateOfAllGames"] = true;
```

Comment:
I'll trust that this is the better place to have this getStateOfAllGames than where was in the first version of this PR.